src/components/features/reviews-section/components/review-form.tsx

```
@@ -0,0 +1,187 @@
+        }
+
+        // Execute the review mutation
+        reviewMutation.mutate(data, {
+            // Success callback
+            onSuccess: () => {
+                toast.success(t("toast.success"));
+                form.reset();
+                setRating(0);
+            },
+            // Error callback
+            onError: (err: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)